common.go

```diff
@@ -98,3 +102,23 @@ func (a ACL) And(session ...proto.SessionType) ACL {
 func (t ACL) Includes(session proto.SessionType) bool {
 	return t&ACL(1<<session) != 0
 }
+
+// VerifyACL checks that the given ACL config is valid for the given service.
+// It can be used in unit tests to ensure that all methods are covered.
+func VerifyACL[T any](acl Config[ACL]) error {
+	var t T
+	iType := reflect.TypeOf(&t).Elem()
+	service := iType.Name()
+	m, ok := acl[service]
+	if !ok {
+		return errors.New("service " + service + " not found")
+	}
+	var errList []error
+	for i := 0; i < iType.NumMethod(); i++ {
+		method := iType.Method(i)
+		if _, ok := m[method.Name]; !ok {
+			errList = append(errList, errors.New(""+service+"."+method.Name+" not found"))
+		}
+	}
+	return errors.Join(errList...)
+}
```

Comment:
This is IMHO much easier to read, much more understandable and without reflect.
Please use this approach.

Then in tests, you just pass this value: https://github.com/0xsequence/builder/blob/master/api/proto/builder.gen.go#L1794

```suggestion
// VerifyACL checks that the given ACL config is valid for the given service.
// It can be used in unit tests to ensure that all methods are covered.
func (acl Config[any]) VerifyACL(webrpcServices map[string][]string) error {
	var errList []error
	for service, methods := range webrpcServices {
		for _, method := range methods {
			if _, ok := acl[service][method]; !ok {
				errList = append(errList, fmt.Errorf("%s.%s not found", service, method))
			}
		}
	}

	return errors.Join(errList...)
}
```

Reply:
This sounds like a good suggestion.